src/main/java/ch/epfl/biop/ij2command/Rois2Labels.java

```diff
@@ -37,26 +40,70 @@
     @Override
     public void run() {
 
-        // reset RoiManager
+        // RoiManager
         RoiManager rm = RoiManager.getRoiManager();
```

Comment:
Why not as as input Parameter this time ?

Reply:
it is , forgot to remove this lien